pkg/mcproducer/templates/machine-config.gotmpl

```diff
@@ -34,7 +34,7 @@ spec:
             [Install]
             WantedBy=multi-user.target
           enabled: true
-          name: "replace-kernel-module.service"
+          name: "{{.KernelModule}}-replace-kernel-module.service"
```

Comment:
I thought we would be using the MachineConfig name as the prefix... WDYT?

Reply:
> I thought we would be using the MachineConfig name as the prefix... WDYT?

@yevgeny-shnaidman  but the override can happen only when we try more than 1 kmod, not? like it says on #920

Reply:
fixed

Reply:
@TomerNewman have you run it on a test cluster?

Reply:
> @TomerNewman have you run it on a test cluster?

I ran unit tests + ran it locally like we did on the meeting.
I will try then to ran it on a cluster additionally

Reply:
the test is to create a simple container image with simple kmod, create the yaml, and then apply it to a cluster and see of the kernel module is loaded into the kernel

Reply:
done, the module was loaded into the kernel at my test cluster